templates/edit_user.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html'%} {% block content %}
+<section class="container">
+  <h2>Edit User Form</h2>
+  <form class="form-group" action="/{{user.id}}/edit" method="POST">
+    <p>
+      First Name
+      <input
+        type="text"
+        placeholder=" Enter a First Name"
+        name="first-name"
+        value="{{user.first_name}}"
+      />
+    </p>
+    <p>
+      Last Name
+      <input
+        type="text"
+        placeholder="Enter a Last Name"
+        name="last-name"
+        value="{{user.last_name}}"
+      />
+    </p>
+    <p>
+      Image-Url
+      <input
+        type="url"
+        placeholder="Provide Image Url"
+        name="img-url"
+        value="{{user.img_url}}"
+      />
+    </p>
+
+    <button class="btn btn-success">Edit</button>
+  </form>
+
+  <form action="/">
```

Comment:
I would clear the form with JavaScript instead of redirecting user when they click on 'cancel'. Being redirected can be too surprising for the user here.